### ISSUE TITLE
eror eval()'d code on line 18

### ISSUE THREAD

**Issue by ezazulian** [allanaes/agenda#2]
cara mengatasinya gimana ya ini eror 

Unhandled Exception

Message:

syntax error, unexpected 'yield' (T_YIELD)
Location:

C:\xampp\htdocs\agenda\laravel\view.php(354) : eval()'d code on line 18

tapi migrate database mah sudah berhasil tingga eror itu ?

**Comment by allanaes** [comment]
Maaf baru bales.. Itu error karena versi PHP yg digunakan adalah versi 5.5
ke atas...

Aplikasi ini menggunakan framework Laravel 3.. Nah, di Laravel 3 itu ada
custom function "yield". Terus, saat rilis PHP 5.5, muncul fitur baru
"yield". Akibatnya, yield dari Laravel 3 yg udah ada lebih dulu jadi
tabrakan dengan yieldnya PHP 5.5.

Solusinya, porting aplikasi ke versi Laravel terbaru (butuh banyak effort)
atau pake PHP versi maksimal 5.4.

Kalau pake bundle WAMP, pastiin downloadnya yg versi 5.4 untuk PHP-nya..

**Comment by ezazulian** [comment]
oh gitu iya udah di downgrade versi PHPnya baru bisa jalan :)
boleh minta emailnya mas .. mau ngomong private :D

**Comment by allanaes** [comment]
Iya, versi PHP-nya mentok di v5.4, soalnya framework yang dipake juga masih Laravel 3 yang dirilis tahun 2012.. :)
Lihat aja di folder migration, di situ kelihatan tanggal tiap dibuat migration untuk update ke databasenya :) 
Untuk email udah saya munculkan di profile :)

**Comment by ezazulian** [comment]
mas mau nanya lagi :) .. ini buat ngeliyat hasil laporan surat masuk sama
surat keluarnya di bagian mana yah sama buat print laporan ... adanya buat
print surat masuk sama surat keluar aja .. print laporannya di sebelah mana
ya mas ? :)

2016-03-22 5:37 GMT+07:00 Allan Ahmad Sarip <notifications@github.com>:

**Comment by allanaes** [comment]
Laporan gimana maksudnya? Di menu Settings ada tab Liberation untuk
download CSV datanya, jadi bisa diimpor ke Excel n pivot sendiri sesuai
kebutuhan.
Tapi kalo yg dibutuhkan itu print daftarnya, bisa di menu search, tp query
dlu / buat pencariannya dlu.

**Comment by ezazulian** [comment]
heheh .. maksudnya mau print laporan surat masuk & keluar pertanggal sekian
di bulan sekian gitu mau di print .. itu belum ada ya .. ?

2016-05-24 7:05 GMT+07:00 Allan Ahmad Sarip <notifications@github.com>:
